src/message/action.js

```diff
@@ -917,8 +919,22 @@ function Originate() {
  * @property {String} StateInterface Optional, State interface
  * @augments Action
  */
-function QueueAdd() {
+function QueueAdd(queue, interface, memberName, penalty, stateInterface) {
```

Comment:
Sounds good, please see my new PR. I took the liberty to rename "interface" in some other places as well.